pom.xml

```diff
@@ -757,12 +757,6 @@
                 </plugin>
                 <!-- maven deploy (End) -->
 
-                <plugin>
```

Comment:
👍 remove the duplicate maven plugin config.

this is the cause of https://github.com/apache/incubator-seatunnel/issues/989, acked.